unittests/Utils/BitVectorSetTest.cpp

```diff
@@ -152,7 +152,7 @@ TEST(BitVectorSet, twoSets) {
   EXPECT_EQ(B.count(2), 0U);
   EXPECT_EQ(B.count(3), 0U);
 }
-
+// Failing test case
```

Comment:
What is falling here and shouldn't we fix this before merging?

Reply:
That's actually fixed by now.